flight/Modules/System/systemmod.c

```diff
@@ -80,6 +80,7 @@ static uint32_t idleCounterClear;
 static struct pios_thread *systemTaskHandle;
 static struct pios_queue *objectPersistenceQueue;
 static bool stackOverflow;
+static uint8_t stackOverflowedName[8];
```

Comment:
similarly can we use the constant that is SYSTEMALARMS_STACKOVERFLOWCODE_SIZE instead of 8